yt_dlp/extractor/niconico.py

```diff
@@ -507,7 +508,7 @@ def get_video_info_xml(items):
 
         thumbnail = (
             self._html_search_regex(r'<meta property="og:image" content="([^"]+)">', webpage, 'thumbnail data', default=None)
-            or get_video_info_web(['thumbnail_url', 'largeThumbnailURL', 'thumbnailURL'])
+            or try_get(get_video_info_web("thumbnail"), ["ogp", "player", "largeUrl", "middleUrl", "url"]) # choose highest from 720p to 240p
```

Comment:
I will merge this for now, but the correct way is to get all these thumbnails seperately and save it in the `thumbnails` keyword so that they are all available in `--list-thumbnails`

Reply:
Oh, I will investigate when I have time